### PR TITLE
Redesign the site using jupyterebook

### PR DESCRIPTION
**Pull request by seisman**
**PREVIEW**: https://seismo-learn.org/sitepreview/seismo-learn/seismology101/jupyterbook/

In this PR, I try to migrate the who documentation from Sphinx to [JupyterBook](https://jupyterbook.org/). 

JupyterBook is built on top of Sphinx, so you can think that it's a wrapper of Sphinx.

Advantages of JupyterBook:

- The default theme is the `sphinx-book-theme`, which looks pretty good
- It supports writing documentation in Markdown, which syntax is easier than ReST.
- It has built-in support for copy-button, panels, and dropdowns. In Sphinx, we have to use many extensions to have these features.
- It also has more feature like "side notes", which is better than dropdowns in some cases.
  image

Disadvantages of JupyterBook:

- Some texts like "Previous" and "Next" are still English-only.

### CODE REVIEW

**Comment by seisman** [comment]
I've migrated the first two parts from Sphinx (ReST) to JupyterBook (Markdown). I think the migration is pretty smooth, and expect to finish the remaining chapters in less than one hour.

@core-man Please see if you agree we should migrate to JupyterBook.

**Comment by seisman** [comment]
Ping @core-man for review.

**Comment by core-man** [comment]
> Ping @core-man for review.

I've finished reviewing the new structure, and am reviewing the content now. I will finish it tomorrow.

File: source/_config.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Book settings
+# Learn more at https://jupyterbook.org/customize/config.html
+
+title: "地震“学”科研入门教程"
+author: seismo-learn
+copyright : "2020–2022"
+#logo: logo.png
```
Comment (core-man) [inline source/_config.yml:7]: Where shall we use logo.png? I see that a new figure is added.

File: source/exercises/introduction.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
-.. figure:: workflow.jpg
-   :alt: 地震学观测的获取和处理流程示意图
-   :width: 95%
-   :align: center
-
-   地震学观测的获取和处理流程示意图。
+:::{figure} workflow.jpg
+:align: center
+:alt: "\u5730\u9707\u5B66\u89C2\u6D4B\u7684\u83B7\u53D6\u548C\u5904\u7406\u6D41\u7A0B\
+:  \u793A\u610F\u56FE"
```
Comment (core-man) [inline source/exercises/introduction.md:21]: Is it wrong?

**Comment by seisman** [comment]
> > Ping @core-man for review.
> 
> I've finished reviewing the new structure, and am reviewing the content now. I will finish it tomorrow.

I don't think you need to review the content, because the conversions from ReST to Markdown are done automatically. If we decide to use jupyterbook, we can merge this PR and then fix any issues later.

File: source/_config.yml (hunk 1 above)
Comment (seisman) [reply source/_config.yml:7]: By default, the logo will be shown like 
image

I'm not sure if we want to have the logo here. I can remove the "logo.png" for now.

Comment (core-man) [reply source/_config.yml:7]: I don't think we want such a logo.

**Comment by core-man** [comment]
> > > Ping @core-man for review.
> > 
> > 
> > I've finished reviewing the new structure, and am reviewing the content now. I will finish it tomorrow.
> 
> I don't think you need to review the content, because the conversions from ReST to Markdown are done automatically. If we decide to use jupyterbook, we can merge this PR and then fix any issues later.

Sounds good~

File: source/exercises/introduction.md (hunk 2 above)
Comment (seisman) [reply source/exercises/introduction.md:21]: Yes, the Chinese figure `alt` attribute is not translated well during the ReST to MarkDown conversion.

File: source/_config.yml (hunk 1 above)
Comment (seisman) [reply source/_config.yml]: Removed.

File: source/exercises/introduction.md (hunk 2 above)
Comment (core-man) [reply source/exercises/introduction.md:21]: We use pandoc to do this conversion, right?

**Review by core-man — APPROVED** [review]

File: source/exercises/introduction.md (hunk 2 above)
Comment (seisman) [reply source/exercises/introduction.md:21]: No, I use this one https://github.com/executablebooks/rst-to-myst.

**Pull request merged**